charts/digger-backend/templates/backend-deployment.yaml

```diff
@@ -35,25 +35,25 @@ spec:
             secretKeyRef:
               name: {{ .Values.postgres.secret.existingSecretName }}
               key: postgres-password
+        {{- else if .Values.digger.postgres.existingSecretName }}
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.digger.postgres.existingSecretName }}
+              key: {{ .Values.digger.postgres.existingSecretKey }}
         {{- else }}
```

Comment:
This conditional applies if postgres is not enabled, which is a bit weird.

Reply:
I think if the "test purposes" postgres is not enabled  (postgres.enabled) then we default to the values of some managed postgres so connection strings are fetched from (digger.postgres.xxxx). I dunno if we should add extra "enabled" flag under digger.postgres since it is already the fallback I guess

Reply:
Oh, sure. That's "built-in postgres enabled". I'm gonna blame the variable name for not being clear, but really it's just that I forgot postgres is always used. 😂

This is fine then. My bad.